os/fs/procfs/fs_procfslogsave.c

```diff
@@ -120,10 +120,6 @@ static int logsave_open(FAR struct file *filep, FAR const char *relpath, int ofl
 {
 	FAR struct logsave_file_s *attr = (struct logsave_file_s *)kmm_zalloc(sizeof(struct logsave_file_s));
 	filep->f_priv = (FAR void *)attr;
-	if (log_dump_read_wake() != OK) {
```

Comment:
Could you let me know why this is removed?

Reply:
This was called whenever we open procfs logsave file which was not needed. We are reading log dump after doing log dump stop . So calling log_dump_read_wake() when log dump stop is called is enough. 

log_dump_read_wake() will start compressing the partially filled log dump buffer . so i dont see why we should call it while opening procfs logsave file.